.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
+    hooks:
+    -   id: check-yaml
+    -   id: end-of-file-fixer
+    -   id: trailing-whitespace
+-   repo: https://github.com/psf/black
+    rev: 19.3b0
+    hooks:
+    -   id: black
```

Comment:
Since the topic of the PR is essentially linting/style, the YAML spacing is inconsistent with `pre-commit.yml`.

Reply:
For reference, I usually do lists like this:

```yaml
repos:
  - stuff: more stuff
    even: more stuff
```

Reply:
That is fixed.